apps/web-client/src/lib/utils/misc.ts

```
@@ -94,7 +94,7 @@ export const mergeBookData = (sources: BookFetchResultEntry[], kind: "prefer_fir
 	!sources.some(Boolean)
 		? undefined
 		: kind === "prefer_first"
-		? sources.reduceRight((acc = {}, curr = {}) => ({ ...acc, ...curr }), {})
-		: sources.reduce((acc = {}, curr = {}) => ({ ...acc, ...curr }), {});
+			? sources.reduceRight((acc = {}, curr = {}) => ({ ...acc, ...curr }), {})
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)